Makefile

```diff
@@ -0,0 +1,35 @@
+# @todo use this APP_CONTAINER in all commands instead of opendorme_app_1
```

Comment:
Here is one pending ToDo - could you remove it if not needed or do what it says?^^

Reply:
unable to figure out a way right now... removed it.. later if needed we will make it dynamic